ckanext/recombinant/i18n/fr/LC_MESSAGES/recombinant.po

```diff
@@ -0,0 +1,277 @@
+# Translations template for ckanext-recombinant.
```

Comment:
should this be the .pot file? Or do you want to submit the translations after this PR?